File: BE/DTOs/QuizWithScoreDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace BE.DTOs
{
    public class QuizWithScoreDTO
    {
        public QuizDTO Quiz { get; set; }
        public int TotalCorrectAnswer { get; set; } = 0;
        public int TotalQuestions { get; set; } = 1;
        public bool IsSubmitted { get; set; } = false;
    }
}

File: FE/app/src/main/java/com/example/tqt_quiz/domain/dto/QuizWithScoreDTO.java
package com.example.tqt_quiz.domain.dto;

import com.google.gson.annotations.SerializedName;

public class QuizWithScoreDTO {
    @SerializedName("quiz")
    QuizDTO quiz;
    @SerializedName("totalCorrectAnswer")
    int totalCorrectAnswer;
    @SerializedName("totalQuestion")
    int totalQuestion;
    @SerializedName("isSubmitted")
    boolean isSubmitted;
    public QuizDTO getQuiz() {
        return quiz;
    }

    public void setQuiz(QuizDTO quiz) {
        this.quiz = quiz;
    }

    public int getTotalCorrectAnswer() {
        return totalCorrectAnswer;
    }

    public void setTotalCorrectAnswer(int totalCorrectAnswer) {
        this.totalCorrectAnswer = totalCorrectAnswer;
    }

    public int getTotalQuestion() {
        return totalQuestion;
    }

    public void setTotalQuestion(int totalQuestion) {
        this.totalQuestion = totalQuestion;
    }

    public boolean isSubmitted() {
        return isSubmitted;
    }

    public void setSubmitted(boolean submitted) {
        isSubmitted = submitted;
    }
}
